--- /Users/Sean/Downloads/xml2rfc-2.4.8-original/changelog	2014-06-05 13:26:55.000000000 -0700
+++ changelog	2014-06-29 20:40:23.000000000 -0700
@@ -1,3 +1,10 @@
+xml2rfc (2.4.8v) ietf; urgency=medium
+
+ * From Sean Leonard <dev+ietf@seantek.com>:
+   Relaxed c element to allow for vspace children, for line breaks
+   inside texttable content. preamble and postamble elements can also
+   have vspace cildren. Issue #262.
+
 xml2rfc (2.4.8) ietf; urgency=medium
 
   * From Tony Hansen <tonyh@att.com>:
