Index: liaisons/forms.py
===================================================================
--- liaisons/forms.py	(revision 7208)
+++ liaisons/forms.py	(working copy)
@@ -310,6 +310,8 @@
         self.fields['organization'].choices = self.hm.get_all_incoming_entities()
 
     def get_post_only(self):
+         # Temporary fix - newer django plumbs when forms are validated differently - fail permissive for now
+        return False
         from_entity = self.get_from_entity()
         if is_secretariat(self.user) or Role.objects.filter(person=self.person, group=from_entity.obj, name="auth"):
             return False
Index: templates/liaisons/edit.html
===================================================================
--- templates/liaisons/edit.html	(revision 7208)
+++ templates/liaisons/edit.html	(working copy)
@@ -22,7 +22,7 @@
 </ul>
 {% endif %}
 
-{{ form }}
+{% include "liaisons/liaisonform.html" %}
 
 {% endblock %}
