Index: ietf/idrfc/views_ballot.py
===================================================================
--- ietf/idrfc/views_ballot.py	(revision 2463)
+++ ietf/idrfc/views_ballot.py	(working copy)
@@ -52,7 +52,7 @@
     return (pos, discuss, comment)
 
 class EditPositionForm(forms.Form):
-    position = forms.ChoiceField(choices=BALLOT_CHOICES, widget=forms.RadioSelect)
+    position = forms.ChoiceField(choices=BALLOT_CHOICES, widget=forms.RadioSelect, required=False)
     discuss_text = forms.CharField(required=False, widget=forms.Textarea)
     comment_text = forms.CharField(required=False, widget=forms.Textarea)
 
@@ -95,7 +95,10 @@
                 setattr(pos, vote, 1)
 
             if pos.id:
-                pos.save()
+                if vote:
+                    pos.save()
+                else:
+                    pos.delete()
                 if vote != old_vote:
                     add_document_comment(request, doc, "[Ballot Position Update] Position for %s has been changed to %s from %s" % (pos.ad, position_label(vote), position_label(old_vote)))
             elif vote:
Index: ietf/idrfc/tests.py
===================================================================
--- ietf/idrfc/tests.py	(revision 2463)
+++ ietf/idrfc/tests.py	(working copy)
@@ -365,7 +365,16 @@
         self.assertTrue(pos.discuss == -1)
         self.assertEquals(draft.idinternal.comments().count(), comments_before + 1)
         self.assertTrue("Position" in draft.idinternal.comments()[0].comment_text)
+        
+        # clear vote
+        comments_before = draft.idinternal.comments().count()
+        r = self.client.post(url, dict(position=""))
+        self.assertEquals(r.status_code, 302)
 
+        pos = Position.objects.filter(ballot=draft.idinternal.ballot, ad__login_name="rhousley")
+        self.assertEquals(len(pos), 0)
+        self.assertEquals(draft.idinternal.comments().count(), comments_before + 1)
+        self.assertTrue("Position" in draft.idinternal.comments()[0].comment_text)
     def test_edit_position_as_secretary(self):
         draft = InternetDraft.objects.get(filename="draft-ietf-mipshop-pfmipv6")
         url = urlreverse('doc_edit_position', kwargs=dict(name=draft.filename))
