From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Mon, 29 Oct 2018 11:08:15 -0400
Subject: fix spelling of "dont"

---
 xml2rfc/run.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xml2rfc/run.py b/xml2rfc/run.py
index 681b968..135b1ca 100755
--- a/xml2rfc/run.py
+++ b/xml2rfc/run.py
@@ -114,7 +114,7 @@ def main():
     plain_options.add_option('-r', '--remove-pis', action='store_true', default=False,
                             help='Remove XML processing instructions')
     plain_options.add_option('-q', '--quiet', action='store_true',
-                            help='dont print anything')
+                            help='don\'t print anything')
     plain_options.add_option('-u', '--utf8', action='store_true',
                             help='generate utf8 output')
     plain_options.add_option('-v', '--verbose', action='store_true',
