--- settings.py	2007-07-13 09:37:48.000000000 -0500
+++ 2.03/ietf/settings.py	2007-07-13 11:39:58.000000000 -0500
@@ -28,13 +28,17 @@
 
 MANAGERS = ADMINS
 
-DATABASE_ENGINE = 'mysql'          #Production Value
-DATABASE_NAME = 'ietf'             #Production Value
-DATABASE_USER = 'django'           #Production Value
-DATABASE_PASSWORD = 'sqlpass!23'   #Production Value
-DATABASE_PORT = ''             	   # Production Value
-#DATABASE_HOST = '10.30.47.10' 	   # The MySQL 5 server, new and hot.. but seems to be slow
-DATABASE_HOST = '10.31.47.10'      # Existing MySQL 4.1 server.. 
+
+DATABASE_ENGINE = 'mysql'           # 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' or 'ado_mssql'.
+
+#WILL NOT WORK IN PRODUCTION
+DATABASE_NAME = 'django_idtracker'             # Or path to database file if using sqlite3.
+DATABASE_USER = 'django'             # Not used with sqlite3.
+DATABASE_PASSWORD = 'playing'         # Not used with sqlite3.
+DATABASE_HOST = ''             # Set to empty string for localhost. Not used with sqlite3.
+DATABASE_PORT = ''             # Set to empty string for default. Not used with sqlite3.
+###########################
+TIME_ZONE = 'America/Los_Angeles'  #SHOULDN'T THIS BE America/New_York ??
 
 
 # Local time zone for this installation. Choices can be found here:
@@ -42,7 +46,7 @@
 # although not all variations may be possible on all operating systems.
 # If running in a Windows environment this must be set to the same as your
 # system time zone.
-TIME_ZONE = 'America/New_York'		#Production Value
+
 
 # Language code for this installation. All choices can be found here:
 # http://www.w3.org/TR/REC-html40/struct/dirlang.html#langcodes
@@ -53,7 +57,7 @@
 
 # If you set this to False, Django will make some optimizations so as not
 # to load the internationalization machinery.
-USE_I18N = True
+USE_I18N = False
 
 # Absolute path to the directory that holds media.
 # Example: "/home/media/media.lawrence.com/"
@@ -92,6 +96,7 @@
 #    'ietf.middleware.PrettifyMiddleware',
     'ietf.middleware.SQLLogMiddleware',
     'ietf.middleware.SMTPExceptionMiddleware',
+    'ietf.middleware.RedirectTrailingPeriod',
     'django.middleware.transaction.TransactionMiddleware',
 )
 
@@ -117,6 +122,7 @@
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.sites',
+    'django.contrib.sitemaps',
     'django.contrib.admin',
     'django.contrib.humanize',
     'ietf.agenda',
@@ -157,7 +163,13 @@
 # The name of the method to use to invoke the test suite
 TEST_RUNNER = 'ietf.tests.run_tests'
 
-IPR_DOCUMENT_PATH = '/home/local/ftp/data/ietf/IPR'   #Production Value
+#I thought I had changed this and committed to SVN
+
+IPR_DOCUMENT_PATH = '/home/master-site/ftp/data/ietf/IPR'
+
+
+#Not sure what this is for.
+TEST_REFERENCE_URL_PREFIX = 'http://compost.research.att.com/old/'
 
 # Put SECRET_KEY in here, or any other sensitive or site-specific
 # changes.  DO NOT commit settings_local.py to svn.
