From: Scott Kitterman <scott@kitterman.com>
Date: Wed, 20 Jun 2018 18:53:57 -0400
Subject: Fix version check to include python3

---
 xml2rfc/run.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xml2rfc/run.py b/xml2rfc/run.py
index 90a89c6..c205b5c 100755
--- a/xml2rfc/run.py
+++ b/xml2rfc/run.py
@@ -334,9 +334,9 @@ def main():
 if __name__ == '__main__':
 
     major, minor = sys.version_info[:2]
-    if not major == 2 and minor >= 6:
+    if not (major == 2 and minor >= 6) and not major == 3:
         print ("")
-        print ("The xml2rfc script requires python 2, with a version of 2.6 or higher.")
+        print ("The xml2rfc script requires python 2, with a version of 2.6 or higher or python 3.")
         print ("Can't proceed, quitting.")
         exit()
 
