--- html.py	2015-05-17 18:52:43.552465200 +0100
+++ /projects/ietf/xml2rfc/trunk/cli/xml2rfc/writers/html.py	2015-05-15 08:34:59.416444700 +0100
@@ -250,16 +250,13 @@
                 return [a]
         elif element.tag == 'eref':
             target = element.attrib.get('target', '')
-            if element.text:
-                a = E.A(element.text, href=target)
+            text = element.text or target
+            if text:
+                a = E.A(text, href=target)
                 a.tail = element.tail
+#                cite = E.CITE('[' + target + ']', title='NONE')
+#                current.append(cite)
                 return [a]
-            else:
-                sp1 = E.SPAN('<')
-                a = E.A(target, href=target)
-                sp2 = E.SPAN('>')
-                sp2.tail = element.tail
-                return [sp1, a, sp2]
         elif element.tag == 'cref':
             self.cref_counter += 1
             anchor = element.attrib.get('anchor', None)
