--- raw_txt.py	2015-05-17 18:52:43.860684100 +0100
+++ /projects/ietf/xml2rfc/trunk/cli/xml2rfc/writers/raw_txt.py	2015-05-17 18:52:26.527800600 +0100
@@ -435,13 +435,11 @@
             elif element.tag == 'eref':
                 if element.text:
                     line.append(element.text + ' ')
-                    self.eref_count += 1
-                    if element.text != element.attrib['target']:
-                        line.append('[' + str(self.eref_count) + ']')
-                        if self.indexmode:
-                            self.eref_list.append([self.eref_count, element])
-                else:
-                    line.append('<' + element.attrib['target'] + '>')
+                self.eref_count += 1
+                if element.text != element.attrib['target']:
+                    line.append('[' + str(self.eref_count) + ']')
+                    if self.indexmode:
+                        self.eref_list.append([self.eref_count, element])
             elif element.tag == 'iref':
                 self._add_iref_to_index(element)
             elif element.tag == 'cref' and \
